packages/front-end/components/Tags/TagsFilter.tsx

```diff
@@ -64,23 +63,7 @@ export default function TagsFilter({
     return (counts[b] || 0) - (counts[a] || 0);
   });
 
-  if (!tags.length && !availableTags.length) {
-    return null;
-  }
-
-  if (!open && !tags.length) {
```

Comment:
I removed this purely from a UX perspective - instead of showing the text, and then having the user select from the dropdown (once visible) I updated the logic so if there were no availableTags to sort through, the dropdown isn't shown.

I'm happy to put it back in if it's a pattern we want to retain.